src/webapp/src/components/interactions/Interactions.vue

```diff
@@ -66,15 +49,13 @@
 import { mapGetters } from "vuex";
 import axios from "axios";
 import InteractionData from "./InteractionData";
-import InteractionDetails from "./interactionDetails.vue/InteractionDetails";
```

Comment:
Should we also remove related Vue components like Chat.vue and InteractionDetails.vue if they are no longer used?